hacks/falco/build_falco.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Build falco with increased ring buffer size for the eBPF probe.
+# https://github.com/ossf/package-analysis/issues/10
+
+FALCO_VERSION=0.27.0
+REV="-3"
+DOCKER_IMAGE=gcr.io/ossf-malware-analysis/falco:$FALCO_VERSION$REV
+FALCO_GIT=https://github.com/falcosecurity/falco
```

Comment:
Should it be from the master/default branch? To have reproducibility we could the pin SHA and pull that to build. This will give us more predictable builds IMO. Thoughts?

Reply:
We checkout the release tag here, which should be almost equally predictable, with the benefit of being less error prone when we upgrade.

Reply:
I agree with that. But I think GitHub release tags can be moved 

```
# assuming you are in the branch referencing currently the right new commit:
git tag -f -a <tagname>

# push your new commit:
git push 

# force push your moved tag:
git push origin -f <tagname>
```